### PR TITLE
Update header and add link

### PR DESCRIPTION
**Pull request by unDemian**
**Built on top https://github.com/Automattic/wp-calypso/pull/20979**

## Summary
- Change title to "WordPress.com Business Concierge Session" (instead of WordPress.com Business Site Setup")
- Add the "Learn more" link which points to https://en.support.wordpress.com/concierge-support/

Mobile view: https://cloudup.com/cwR2-x0WEKA
Desktop view: https://cloudup.com/cM4nyUEciqA

### CODE REVIEW

**Comment by unDemian** [comment]
Thanks for taking the time, I pushed some updates to address your suggestions. I also updated the description with screenshots.

File: client/me/concierge/primary-header.js [hunk 1]
```diff
@@ -10,22 +10,36 @@ import React, { Component } from 'react';
...
+					headerText={ translate( 'WordPress.com Business Concierge Session' ) }
+					subHeaderText={ translate(
+						"In this 30 minute session we'll help you get started with your site."
+					) }
 				/>
+				<ExternalLink
+					className="concierge__info-link"
+					icon={ false }
+					href={ 'https://en.support.wordpress.com/concierge-support' }
+					rel="noopener noreferrer"
```
[the remark this answers is not in the document]
Comment (unDemian) [reply client/me/concierge/primary-header.js]: Nice, updated.

File: client/me/concierge/primary-header.js [hunk 2]
```diff
@@ -10,22 +10,36 @@ import React, { Component } from 'react';
...
-					subHeaderText="In this 30 minute session we'll help you get started with your site."
+					headerText={ translate( 'WordPress.com Business Concierge Session' ) }
+					subHeaderText={ translate(
+						"In this 30 minute session we'll help you get started with your site."
+					) }
 				/>
+				<ExternalLink
+					className="concierge__info-link"
+					icon={ false }
+					href={ 'https://en.support.wordpress.com/concierge-support' }
```
[the remark this answers is not in the document]
Comment (unDemian) [reply client/me/concierge/primary-header.js]: Nice, I didn't know of it, updated to use the library.

**Comment by omarjackman** [comment]
Thanks for the screenshots @unDemian. It looks like theres a bit of extra space between the "Learn more" and the text above it. You might want to put zero padding/margin on the `.concierge__info-link` so that theres consistent spacing around the blocks of text.

**Comment by unDemian** [comment]
The link element doesn't have any margin/paddings https://cloudup.com/cBt6kAICJEV, the extra space comes from the margin of the `FormattedHeader` component, I guess I could overwrite its styles.

**Comment by omarjackman** [comment]
> the extra space comes from the margin of the `FormattedHeader` component

Well you have a few options. Maybe you should add the `ExternalLink` to the `subHeaderText`.

Something like this might work:

```jsx
const subHeaderText = (
    <div>
        { translate(
            "In this 30 minute session we'll help you get started with your site."
        ) }
        <ExternalLink
            className="concierge__info-link"
            icon={ false }
            href={ support.CONCIERGE_SUPPORT }
            target="_blank">
            { translate( 'Learn more' ) }
        </ExternalLink>
    </div>
);

...

subHeaderText={ subHeaderText }
```

**Comment by unDemian** [comment]
Nice, now it looks something like https://cloudup.com/cXBeWcoVPcS

**Comment by omarjackman** [comment]
> Nice, now it looks something like https://cloudup.com/cXBeWcoVPcS

cool, now lets maybe add 5px to the top of the link so that it looks like the original design

32344113-0752ab5a-c006-11e7-82c9-8b88dec53d9f

**Comment by unDemian** [comment]
Actually embedding the `ExternalLink` into the`subHeaderText` will trigger a validation error. 

In order to achieve a pixel perfect match with the design I'd have to overwrite the margins of both formatted header div and it's paragraph or use negative margins for the link. For the sake of product design consistency I will not do that. Giving the fact that this PR is blocking I will just merge the default formatted version https://cloudup.com/cMc4Xft6ph8

@omarjackman @evilluendas if this turns out to be a big deal, we could create a different ticket and add some custom formatting to match the design

**Pull request merged**